pkg/cloudprovider/providers/gce/gce_loadbalancer_external.go

```diff
@@ -991,15 +987,28 @@ func (gce *GCECloud) getServiceNetworkTier(svc *v1.Service) (NetworkTier, error)
 	return tier, nil
 }
 
-// reconcileFwdRuleNetworkTiers check the network tiers of existing forwarding
+func (gce *GCECloud) deleteWrongNetworkTieredResources(lbName, lbRef string, desiredNetTier NetworkTier) error {
+	if !gce.AlphaFeatureGate.Enabled(AlphaFeatureNetworkTiers) {
```

Comment:
Humm..Seems like the feature gate check is moved inside functions. I think having it outiside may make it clear about what are involved.

Reply:
Restored it.